modules/UI/videolayout/SmallVideo.js

```diff
@@ -556,10 +558,39 @@ export default class SmallVideo {
         }
 
         if (this.displayMode !== oldDisplayMode) {
+            this.reArrangeVideos();
             logger.debug(`Displaying ${displayModeString} for ${this.id}, data: [${JSON.stringify(displayModeInput)}]`);
         }
     }
 
+    /**
+     * Change the order of the element, based on displayMode
+     * If is sharing screen/streaming order should be < 10
+     * Else order > 10
+     * Also adds special className 'without-camera' to apply
+     * different styles (smaller width and height)
+     */
+    reArrangeVideos() {
+        if (!this.isLocal
+            && (this.displayMode === DISPLAY_VIDEO || this.displayMode === DISPLAY_VIDEO_WITH_NAME)) {
+            this.container.style.order = 1;
+            this.$container.removeClass('without-camera');
+        } else if (!this.isLocal
+            && (this.displayMode === DISPLAY_AVATAR_WITH_NAME || this.displayMode === DISPLAY_AVATAR)) {
+            this.container.style.order = 11;
+            this.$container.addClass('without-camera');
```

Comment:
not important really, but we usually prefer to use `toggleClass('class', true)` and `toggleClass('class', false)` which is easier to maintain later

Reply:
fixed

Reply:
In fact you can really simplify that whole block. Put `order:1` /`order:11` into css, and just toggle 2 classes: 1st that is "bring on stage"(order 1), second that is "bring back" (order: 11) and one for "default" (probably no class, no order)

Reply:
Though it seems acceptable now

Reply:
Im wondering why i didn't do it this way initially. Will fix